examples/aarch64-softmmu/qemuconf.json

```diff
@@ -0,0 +1,8 @@
+{
+	"qemu" : "../../qemu/build/debug/qemu-system-aarch64",
+	"bios" : "",
+	"kernel" : "minimal.elf",
+	"plugin" : "../../faultplugin/libfaultplugin.so",
+	"machine" : "stm32vldiscovery",
```

Comment:
This will create an ARM machine. For aarch64 we could for example use machine `virt` and specify the CPU in `additional_qemu_args` as `-cpu cortex-a53` (this is the config from the aarch64-virt buildroot config).